runtime/src/main/java/io/quarkiverse/ironjacamar/runtime/endpoint/TransactionAwareMessageEndpoint.java

```diff
@@ -31,7 +30,7 @@ public void beforeDelivery(Method method) throws ResourceException {
         Arc.container().requestContext().activate();
         QuarkusTransaction.begin();
         try {
-            Transaction transaction = TransactionManager.transactionManager().getTransaction();
+            Transaction transaction = Arc.container().instance(TransactionManager.class).get().getTransaction();
```

Comment:
Isn't the same object returned?

Reply:
No, it is different. There is [NotifyingTransactionManager](https://github.com/quarkusio/quarkus/blob/main/extensions/narayana-jta/runtime/src/main/java/io/quarkus/narayana/jta/runtime/NotifyingTransactionManager.java) from Arc Container which can add some CDI events processing. I think it chould be better to consistent with the same implementation in `quarkus-narayana-jta` extension.